project/backend/doc-server/src/main/resources/files/dd398660-ca95-4b58-97c8-5c7c5d2a8a31.py

```
@@ -0,0 +1,128 @@
+api_key = os.getenv('gemini_api_key')
+url = "https://generativelanguage.googleapis.com/v1beta/models/gemini-1.5-flash-latest:generateContent"
+
+app = Flask(__name__)
+CORS(app)
+
+# Utility functions
+def split_document(doc, chunk_size=5):
+    sentences = sent_tokenize(doc)
+    return [' '.join(sentences[i:i + chunk_size]) for i in range(0, len(sentences), chunk_size)]
+
+def generate_embeddings(chunks):
```

Ruff (0.8.2)

Missing return type annotation for public function `generate_embeddings`

(ANN201)